compiler/src/dmd/declaration.h

```diff
@@ -272,6 +272,12 @@ class VarDeclaration : public Declaration
     bool isArgDtorVar(bool v);
     bool isCmacro() const; // if a C macro turned into a C variable
     bool isCmacro(bool v);
+#if MARS
+    bool inClosure() const; // is inserted into a GC allocated closure
+    bool inClosure(bool v);
+    bool inAlignSection() const; // is inserted into aligned section on stack
+    bool inAlignSection(bool v);
+#endif
```

Comment:
Since these are mars only I don't think you need to put these in the headers.

Reply:
You never know...  Also, we should test these additions in the cxxfrontend test.

Reply:
Why?